pkgs/development/libraries/openssl/default.nix

```diff
@@ -34,9 +34,20 @@ let
       substituteInPlace crypto/async/arch/async_posix.h \
         --replace '!defined(__ANDROID__) && !defined(__OpenBSD__)' \
                   '!defined(__ANDROID__) && !defined(__OpenBSD__) && 0'
+    '' + optionalString (versionAtLeast version "1.1.1" && static)
```

Comment:
Why does this depend on 1.1.1?

Reply:
It should probably also be good for 1.0.2. I'll test that and remove the check if it works!